ci-operator/step-registry/baremetal/lab/post/wipe/baremetal-lab-post-wipe-ref.yaml

```diff
@@ -14,6 +14,8 @@ ref:
       default: ""
     - name: SELF_MANAGED_NETWORK
       default: "true"
+    - name: BOOT_MODE
```

Comment:
Remove as not needed and confusional with respect to other BM profiles (e.g., PXE UPI on BM)

Reply:
Done